presto-native-execution/src/test/java/com/facebook/presto/nativeworker/ContainerQueryRunnerUtils.java

```diff
@@ -0,0 +1,284 @@
+
+    static Path createTpchProperties()
+            throws IOException
+    {
+        Path tempTpchProperties = Files.createTempFile("tpch", ".properties");
+        String properties = "connector.name=tpch\n" +
+                "tpch.column-naming=STANDARD\n";
+        Files.write(tempTpchProperties, properties.getBytes(), StandardOpenOption.APPEND);
+        return tempTpchProperties;
+    }
+
+    static Path createNativeWorkerConfigProperties(int coordinatorPort)
+            throws IOException
+    {
+        Path tempConfigProperties = Files.createTempFile("config", ".properties");
+        String properties = "presto.version=testversion\n" +
+                "http-server.http.port=7777\n" +
+                "discovery.uri=http://presto-coordinator:" + coordinatorPort + "\n" +
+                "system-memory-gb=2\n" +
+                "native.sidecar=false\n";
+        Files.write(tempConfigProperties, properties.getBytes(), StandardOpenOption.APPEND);
+        return tempConfigProperties;
+    }
+
+    static Path createCoordinatorConfigProperties(int port)
+            throws IOException
+    {
+        Path tempConfigProperties = Files.createTempFile("config", ".properties");
+        String properties = "coordinator=true\n" +
+                " presto.version=testversion\n" +
+                " node-scheduler.include-coordinator=false\n" +
+                " http-server.http.port=" + port + "\n" +
+                " discovery-server.enabled=true\n" +
+                "discovery.uri=http://presto-coordinator:" + port + "\n" +
+                " native-execution-enabled=true\n" +
+                " optimizer.optimize-hash-generation=false\n" +
+                " parse-decimal-literals-as-double=true\n" +
+                " use-alternative-function-signatures=true\n" +
+                " experimental.table-writer-merge-operator-enabled=false\n" +
+                " inline-sql-functions=false\n" +
+                " offset-clause-enabled=true\n" +
+                " regex-library=RE2J\n";
+        Files.write(tempConfigProperties, properties.getBytes(), StandardOpenOption.APPEND);
+        return tempConfigProperties;
+    }
+
+    static Path createCoordinatorJvmProperties()
+    {
+        try {
+            Path tempConfigProperties = Files.createTempFile("jvm", ".properties");
+            String properties = "-server\n" +
+                    "-Xmx1G\n" +
+                    "-XX:+UseG1GC\n" +
+                    "-XX:G1HeapRegionSize=32M\n" +
+                    "-XX:+UseGCOverheadLimit\n" +
+                    "-XX:+ExplicitGCInvokesConcurrent\n" +
+                    "-XX:+HeapDumpOnOutOfMemoryError\n" +
+                    "-XX:+ExitOnOutOfMemoryError\n" +
+                    "-Djdk.attach.allowAttachSelf=true\n";
+            Files.write(tempConfigProperties, properties.getBytes(), StandardOpenOption.APPEND);
+            return tempConfigProperties;
+        }
+        catch (IOException e) {
```

Comment:
Why are similar IOException not caught in the above functions for co-ordinator and worker config as well ?